Gemfile

```diff
@@ -24,11 +24,7 @@ gem 'cancancan', '~> 1.13', '>= 1.13.1'
 gem 'govdelivery-tms', '2.8.4', require: 'govdelivery/tms/mail/delivery_method'
 # Use postgresql as the database for Active Record
 gem 'pg', '~> 0.15'
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rubyzip', '>= 1.3.0'
-
-gem 'sentry-raven', '~> 2.3.0'
+gem 'sentry-raven', '2.9.0' # don't change gem version unless sentry server is also upgraded
```

Comment:
I don't think there's a reason to updade sentry at the same time as rails.  Please do it before or after.

Reply:
This was part of the "it's in vets-api" changes, reverted